frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div>
+    <div class="container-fluid" data-feedback-code-mirror>
+      <textarea v-show="false" ref="cm-editor" v-model="value"></textarea>
+    </div>
+    <div class="container-fluid text-right py-2">
+      <button
+        class="btn btn-primary mx-2"
+        :disabled="!numberOfComments"
+        @click.prevent="saveFeedbackList"
+      >
+        {{ T.submissionSendFeedback }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Ref } from 'vue-property-decorator';
+import T from '../../lang';
+import CodeMirror from 'codemirror';
+import { EditorOptions, languageModeMap, modeList } from './CodeView.vue';
+import Feedback, { ArenaCourseFeedback, FeedbackStatus } from './Feedback.vue';
+
+for (const mode of modeList) {
+  require(`codemirror/mode/${mode}/${mode}.js`);
+}
+
+@Component({
+  components: {},
+})
+export default class FeedbackCodeView extends Vue {
+  @Prop() language!: string;
+  @Prop() value!: string;
+  @Prop({ default: () => new Map() }) feedbackList!: Map<
```

Comment:
```suggestion
  @Prop({ default: () => new Map<number, FeedbackClass>() }) feedbackMap!: Map<
```

esto ya no es una lista.

Reply:
Listo